riscv-debug-spec.tex

```diff
@@ -619,6 +642,34 @@ \subsection{JTAG Connector}
 connector looks as it does in Table~\ref{tab:header}. The function of each pin
 is described in Table~\ref{tab:pinout}.
 
+%MAW: Why do we only have 'RESET', but no 'TRST(n)' signal?
+% See http://openocd.org/doc/html/Reset-Configuration.html
+% Both are useful.
+% One is explicitly ``test logic reset'', which would imply
+% resetting the JTAG state machine and all associated test logic, in this
+% case the Debug Transport Module, synchronizers, and perhaps even
+% the Debug Module itself. It basically does the same thing as getting
+% the JTAG state machine into TEST_LOGIC_RESET state, which is
+% why it is optional.
```

Comment:
Because some circuits may not have a POR circuit to actually reset the test logic.... so if I don't hook up a debugger, somehow that logic has to be reset (the JTAG standard says you can't use the system reset for this).